clang/test/Format/dump-config-objc-stdin.m

```diff
@@ -1,5 +1,8 @@
+// RUN: clang-format -assume-filename=foo.m -dump-config | FileCheck %s
```

Comment:
It looks like you reverted the lit test which just runs `clang-format -dump-config` without any other arguments. Is that what you wanted to do?

Reply:
Yes, because it wouldn't make a difference before and after the patch. I don't think we can use lit to test that behavior (i.e., not waiting for an EOF) of clang-format in the terminal.